.github/workflows/build.yaml

```diff
@@ -241,8 +241,9 @@ jobs:
             output/images/haos_ova*.qcow2.xz
 
   bump_version:
-    name: Bump dev channel version
+    name: Bump ${{ needs.prepare.outputs.channel }} channel version
     if: ${{ github.repository == 'home-assistant/operating-system' }}
+    environment: "${{ github.event_name == 'release' && 'version_release' || '' }}"
```

Comment:
While at it, we could have two environments directly, one for beta and one for stable. So, how about:

```
    environment: "${{ github.event_name == 'release' &&  needs.prepare.outputs.channel || '' }}"
```

Reply:
Sure, updated :+1: